notebooks/class_9_intro_cv_cnns.ipynb

```diff
@@ -525,16 +500,32 @@
    "outputs": [],
    "source": [
     "###If using Binder:\n",
-    "os.makedirs(os.path.join(root,'data','mnist','3.0.1'))\n",
+    "datadir = os.path.join(root,'data','mnist','3.0.1')\n",
```

Comment:
These are the only material changes in the notebook. The rest seems to be stripping an 'id' field from each cell which I could not avert but seems to be innocuous

Reply:
great, thanks for this. Any idea how I can save a notebook without the "id" field?